Dockerfile

```diff
@@ -12,6 +12,8 @@ RUN mkdir -p /usr/share/man/man1 \
     dpkg \ 
     zip \
     default-jre \
+    && curl -fsSL https://deb.nodesource.com/setup_16.x | bash - \
```

Comment:
Any issues with v18?

Reply:
I used the LTS support, which is currently v16, to get a more stable version. Do you want to me switch to v18?

Reply:
No update to README needed, will fix the others.

Reply:
No, I thought 18 was TLS by now :)